webapp/cypress/integration/createBoard.ts

```diff
@@ -112,4 +108,43 @@ describe('Create and delete board / card', () => {
         cy.get('.DeleteBoardDialog button.danger').click({force: true})
         cy.contains(boardTitle).should('not.exist')
     })
+
+    it('MM-T4433 Scrolls the kanban board when dragging card to edge', () => {
+        // Visit a page and create new empty board
+        cy.visit('/')
+        cy.uiCreateBoard('Empty board')
+
+        // Create 10 empty groups
+        cy.log('**Create new empty groups**')
+        for (let i = 0; i < 10; i++) {
+            cy.contains('+ Add a group').scrollIntoView().should('be.visible').click()
+            cy.get('.KanbanColumnHeader .Editable[value=\'New group\']').should('have.length', i + 1)
+        }
+
+        // Create empty card in first group
+        cy.log('**Create new empty card in first group**')
+        cy.contains('+ New').click()
+        cy.get('.Dialog').should('exist')
+        cy.get('.Dialog Button[title=\'Close dialog\']').should('be.visible').click()
+        cy.get('.KanbanCard').should('exist')
+
+        // Drag card to right corner and expect scroll to occur
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.get('.Kanban').invoke('scrollLeft').should('equal', 0).wait(1000)
+
+        // wait necessary to let state change propagate
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.get('.KanbanCard').
+            trigger('dragstart').
+            wait(500)
+
+        // wait necessary to trigger scroll animation for some time
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.get('.Kanban').
+            trigger('dragover', {clientX: Cypress.config().viewportWidth - 100, clientY: Cypress.config().viewportHeight / 2}).
+            wait(1500).
+            trigger('dragend')
+
+        cy.get('.Kanban').invoke('scrollLeft').should('not.equal', 0)
```

Comment:
@venarius Looking at https://github.com/mattermost/focalboard/issues/1731, it's about scrolling behavior from right to left. With that, could you please add another step/check that verifies such scroll direction.

Reply:
@saturninoabril I added another check for the other direction but imho I would advise against adding this extra step as this adds 2 more waiting-seconds and this could stack up a lot on later CI runs when there are more cypress tests.

Reply:
Not sure if you noticed but the added step is not working/scrolling to the left. Maybe because the card is not picked up from the right?
Noted on extra overhead. If going to choose, then I'd recommend to test scroll behavior from right to left, to align with the original issue.

Reply:
yeah, I changed the test to be from right to left 👍